send-surveys/src/main/java/com/objectcomputing/pulsesurvey/receive/responses/SurveyResponseController.java

```diff
@@ -164,12 +165,19 @@ boolean saveResponse(String currentEmotion, String surveyKey) {
         return responseAdded;
     }
 
-    private void saveUserComment(String surveyKey, String comments) {
+    boolean saveUserComment(String surveyKey, String comments) {
```

Comment:
I'm not sure that I understand the need for the boolean return type here.  I think it was fine with void, wasn't it?

Reply:
this makes it easier to test. otherwise I have nothing to compare